File: src/sdk-reference/cpp/1/document/deleteByQuery/snippets/delete-by-query.cpp
try {
  std::vector<std::string> response =
    kuzzle->document->deleteByQuery("nyc-open-data", "yellow-taxi", R"({
      "query": {
        "term": {
          "capacity": 7
        }
      }
    })");

  std::cout << "Successfully deleted " << response.size() << " documents" << std::endl;
} catch (kuzzleio::KuzzleException& e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/document/deleteByQuery/snippets/delete-by-query.cs
try {
  List<string> response =
    kuzzle.document.deleteByQuery("nyc-open-data", "yellow-taxi", @"{
      ""query"": {
        ""term"": {
          ""capacity"": 7
        }
      }
    }");

  Console.WriteLine("Successfully deleted " + response.Count + " documents");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
